testindex/validators.py

```diff
@@ -314,8 +323,21 @@ def run(self):
 
             container_names[container_name].append(entry["id"])
 
+            # * Check for pre-build script
+            # TODO : Make a better implementation of pre-build script checking
+            # TODO : Ideally, if prebuild is not in entry, it wont reach here and this should happen if it is not None
+            prebuild_exists = False
+            if "prebuild-script" in entry and "prebuild-path" in entry:
+                prebuild_exists = True
+                prebuild_script = entry.get("prebuild-script")
+                prebuild_path = entry.get("prebuild-path")
```

Comment:
@mohammedzee1000 can we change `prebuild_path` to `prebuild_context`? this will give the user a better reason for adding the variable

Reply:
@bamachrn sure, i can do that.